### PR TITLE
MongoDB - Batch Size On Aggregate Options & Find Options

### PR DESCRIPTION
**Pull request by mcserra**
Solves #11565

### CODE REVIEW

File: extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/AggregateOptions.java [hunk 1]
```diff
@@ -114,6 +115,17 @@ public AggregateOptions hint(Bson hint) {
         return this;
     }
 
+    /**
+     * Sets the batchSize.
```
Comment (loicmathieu) [inline extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/AggregateOptions.java:119]: Can you improve the JavaDoc to explain how this is used ?
You can check the JavaDoc of `com.mongodb.reactivestreams.client.AggregatePublisher.batchSize(int)` as it's the method being called with this attribute.

File: extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/FindOptions.java [hunk 2]
```diff
@@ -250,6 +251,17 @@ public FindOptions showRecordId(boolean showRecordId) {
         return this;
     }
 
+    /**
+     * Sets the batchSize.
```
Comment (loicmathieu) [inline extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/FindOptions.java:255]: Same remark here for the JavaDoc improvement

**Review by loicmathieu — CHANGES_REQUESTED** [review]
Some feedback around JavaDoc. Please fix them and update the PR, this will trigger a new CI that I hope will pass as tests are failing in the current PR.

File: extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/AggregateOptions.java (hunk 1 above)
Comment (mcserra) [reply extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/AggregateOptions.java]: I don't know if the second part is necessary, but anyways, I copied it.

**Review by loicmathieu — APPROVED** [review]
LGTM

**Comment by loicmathieu** [comment]
The CI check failed is in an un-related area, I'll check with the team that I can merge with it.

**Comment by loicmathieu** [comment]
Merged as CI is failing on a known flawky test.
Thanks

**Pull request merged**